### PR TITLE
Perdona Santi me dormi );

### PR DESCRIPTION
**Pull request by MarcoAlejandroLopezGomez**
Before issuing a pull request, please see the contributing page.

### CODE REVIEW

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/AutonomousCommands/ForwardWhileIntake.java
Comment (SantiQ0905) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/AutonomousCommands/ForwardWhileIntake.java:1]: Make sure to fix this command. Incomplete

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/ChassisPaths.java
Comment (SantiQ0905) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/ChassisPaths.java:1]: Make sure to fix this command. Incomplete

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/ClawMove.java
Comment (SantiQ0905) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/ClawMove.java:1]: Review this file.

**Review by SantiQ0905 — CHANGES_REQUESTED** [review]

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/AutonomousCommands/ForwardWhileIntake.java
Comment (MarcoAlejandroLopezGomez) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/AutonomousCommands/ForwardWhileIntake.java:1]: noted

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/ChassisPaths.java
Comment (MarcoAlejandroLopezGomez) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/ChassisPaths.java:1]: noted

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/ClawMove.java
Comment (MarcoAlejandroLopezGomez) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Commands/ClawMove.java:1]: Will work in that with Paulina

**Review by SantiQ0905 — APPROVED** [review]
Please test the Claw and twist.

**Pull request merged**